utils/log_writer.go

```diff
@@ -0,0 +1,22 @@
+package utils
+
+import (
+	"bufio"
+	"io"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// NewLogWriter pipes stdout/err logs to logrus
+func NewLogWriter(fields log.Fields, level log.Level) io.WriteCloser {
```

Comment:
I'm not sure I fully grasp the issue since the terms we are using are confusing. I'll define what I mean and then try to state what the code is currently doing. If I need to make any changes, let me know using the terms below.

-  **application**: the shimmed application run by ContainerPilot
-  **process**: a process being run by ContainerPilot as part of it's normal operation: health check, task, sensor, etc...
- **log**: refers to the `logrus` logging backend
- **output**: refers to the two native output streams: (`os.Stdout` and `os.Stderr`)

ContainerPilot will always allow the application to stream its logs directly to `stdout` and `stderr` since it attaches the streams directly in. Currently this is done by default in `util.ExecuteAndWait`.

I'm thinking that the behavior for **app** remains the same. In essence, an **app** never writes output to the **log** and thus is unaffected by the log levels in the logging config.

```go
cmd.Stdout = os.Stdout
cmd.Stderr = os.Stderr
``` 

But for a **process** we take a different approach and instead:

```go
stdout := utils.NewLogWriter(fields, log.InfoLevel)
stderr := utils.NewLogWriter(fields, log.DebugLevel)
cmd.Stdout = stdout
cmd.Stderr = stderr
```

This means that all **process** output is shipped to the **log** which can be adjusted with the log config.

I think at this point we just need to agree on which `logrus.Level` **process** output streams should map to by default, and if those streams should be configurable.